src/js/components/Header/VolumeControl.react.js

```diff
@@ -54,7 +54,7 @@ export default class VolumeControl extends Component {
                            min={ 0 }
                            max={ 1 }
                            step={ 0.01 }
-                           defaultValue={ this.state.volume, 1 / 4 }
+                           defaultValue={ Math.pow(this.state.volume, 1 / 4) }
```

Comment:
I might be wrong, but I believe this should go to line 24, so that line 51 is also updated correctly.